pom.xml

```diff
@@ -52,6 +52,8 @@
     <jenkins.baseline>2.504</jenkins.baseline>
     <jenkins.version>${jenkins.baseline}.1</jenkins.version>
     <hpi.compatibleSinceVersion>2.0.0</hpi.compatibleSinceVersion>
+    <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+    <hpi.bundledArtifacts>jbcrypt,tink,trilead-putty-extension,trilead-ssh2</hpi.bundledArtifacts>
```

Comment:
I do not think is a good idea

Reply:
The point of this change is not only to prevent future bugs, but also to make your life easier as a maintainer, by giving you confidence when updating or changing dependencies that you are not accidentally including some library which you do not intend to bundle, which has caused many bugs in the Jenkins ecosystem over the years, and is a relatively easy mistake to make. The hope is that you will never have to change this list or otherwise care about it except if you are actually changing bundled dependencies, which should be infrequent.